dsf-gdb/org.eclipse.cdt.dsf.gdb/src/org/eclipse/cdt/dsf/gdb/launching/FinalLaunchSequence.java

```diff
@@ -134,12 +134,14 @@ protected String[] getExecutionOrder(String group) {
 					//
 					// "stepSetSourceLookupPath",   //$NON-NLS-1$
 
-					// For remote-attach launch only (deprecated)
+					// For remote-attach launch only (deprecated, see javadocs)
 					"stepRemoteConnection", //$NON-NLS-1$
 					// For all launches except attach ones
 					"stepNewProcess", //$NON-NLS-1$
 					// For all attach launch only
 					"stepAttachToProcess", //$NON-NLS-1$
+					// For remote attach launch only (deprecated, see javadocs)
+					"stepAttachRemoteToDebugger", //$NON-NLS-1$
```

Comment:
if we want to keep it then we need `fGDBBackend.getIsAttachSession()` check as well.. previously `stepAttachToProcess` was taking care of it..

Reply:
in fact also need `fGDBBackend.getSessionType() == SessionType.REMOTE` check..